lib/builders/latexmk.js

```diff
@@ -77,12 +80,35 @@ export default class LatexmkBuilder extends Builder {
 
     if (engine.match(latexRegExp)) {
       args.push(`-latex="${engine}"`)
-      if (outputFormat !== 'dvi') {
-        atom.notifications.addWarning(
-          'PS/PDF output is not supported by LaTeX engines.'
-        )
+      if (outputFormat === 'pdf') {
+        switch (producer) {
+          case 'default':
+            // default is an option to protect a user's .latexmkrc from override
+            args.push('-pdfdvi')
+            break
+          case 'dvipdf':
+            args.push('-pdfdvi')
+            args.push('-e "\\$dvipdf = \'dvipdf %O %S %D\';"')
+            break
+          case 'dvipdfm':
```

Comment:
Thanks for the feedback @yudai-nkt!

It looks to me like both TeX Live and MiKTeX currently have `dvipdfmx` symlinked to `xdvipdfmx` so it is probably worth simplifying the list.

Cheers!